backend/orchestrator/function_pb2.pyi

```diff
@@ -166,20 +166,21 @@ class Function(google.protobuf.message.Message):
     KEY_FIELD_NUMBER: builtins.int
     NAME_FIELD_NUMBER: builtins.int
     DESCRIPTION_FIELD_NUMBER: builtins.int
-    FUNCTION_FIELD_NUMBER: builtins.int
+    ARCHIVE_FIELD_NUMBER: builtins.int
```

Comment:
I see changes there but no companion PR linked to an update of the orchestrator. Am I missing something?

Reply:
Thanks for the update - Or sorry if I missed the link in the first place 🙏